ci/run-tests.ps1

```diff
@@ -37,12 +39,10 @@ Write-Log "Running $($ue_path_absolute) $($cmd_args_list)"
 $run_tests_proc = Start-Process -PassThru -NoNewWindow $ue_path_absolute -ArgumentList $cmd_args_list
 Wait-Process -Id (Get-Process -InputObject $run_tests_proc).id
 
-# Workaround for UNR-2156, where spatiald / runtime processes sometimes never close
+# Workaround for UNR-2156 and UNR-2076, where spatiald / runtime processes sometimes never close, or where runtimes are orphaned
 # Clean up any spatiald and java (i.e. runtime) processes that may not have been shut down
-Stop-Process -Name "spatiald" -ErrorAction SilentlyContinue # if no process exists, just keep going
-Stop-Process -Name "java" -ErrorAction SilentlyContinue # if no process exists, just keep going
-
-Write-Log "Exited with code: $($run_tests_proc.ExitCode)" # I can't find any indication of what the exit codes actually mean, so let's not rely on them
```

Comment:
Unreal says a non-zero code is produced if tests fail, but this would need more testing (we check the results of the tests in a different way, anyway).

Reply:
I figured that if we don't rely on those codes anyway, or have proper documentation for them, we might as well drop the `Write-Log` reporting them.